src/main/js/components/timeline/MainTimelineElement.js

```diff
@@ -0,0 +1,19 @@
+import TweetBlock from "./TweetBlock";
+import React, { Component } from "react";
+
+// Display List of Tweets or error message
+export default class MainTimelineElement extends React.Component {
+
+	render() {
+		if (this.props.tweets) {
```

Comment:
check your code logic and ensure you get back what is expected

Reply:
Are you talking about the case where map cannot be performed on the tweets prop?